default.nix

```diff
@@ -1,7 +1,21 @@
-with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/fb235c98d839ae37a639695ad088d19ef8382608.tar.gz) {});
-# NOTE: when bumping nixpkgs, also update nixpkgs-src.json
+let
```

Comment:
can you rename this to `shell.nix`? the constant changes in here are causing merge conflicts in my devops-13 PR

Reply:
OK fixed